db/index.js

```diff
@@ -0,0 +1,82 @@
+let mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/photo-gallery-container');
+
+
+const PhotoSchema = new mongoose.Schema({
+  img: String, /*a link to img on S3 server, assuming I understand how S3 works properly*/
+  caption: String,
+  date: Date, //Will order them by either this
+  helpfulRates: Number, //Or this
+  unhelpfulRates: Number,
+  posterInfo: { //Could also be its own schema?
```

Comment:
Since Mongo is able to store objects without needing its own schema, I think this is fine as it is.